src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/entity/request/LockRequest.kt

```diff
@@ -0,0 +1,13 @@
+package uk.gov.justice.digital.hmpps.sentenceplan.entity.request
+
+import uk.gov.justice.digital.hmpps.sentenceplan.data.UserDetails
+
+data class LockRequest(
+  val lockType: LockType,
+  val userDetails: UserDetails,
+)
```

Comment:
Can we rename this to a `SignRequest`? and a `SignType`

Reply:
Resolved in https://github.com/ministryofjustice/hmpps-sentence-plan/pull/109/commits/3a12e9b9aad988b79609fe6e0b46e5b4e0bacc0e